specs/_features/reuse_indexes/beacon-chain.md

```diff
@@ -0,0 +1,66 @@
+# Reuse indexes -- The Beacon Chain
+
+## Table of contents
+
+<!-- TOC -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Preset](#preset)
+  - [Time parameters](#time-parameters)
+- [Helpers](#helpers)
+  - [Predicates](#predicates)
+    - [`is_reusable_validator`](#is_reusable_validator)
+- [Beacon chain state transition function](#beacon-chain-state-transition-function)
+  - [Block processing](#block-processing)
+    - [Modified `get_index_for_new_validator`](#modified-get_index_for_new_validator)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- /TOC -->
+
+## Introduction
+
+This is the beacon chain specification to assign new deposits to existing validator records that have withdrawn long ago.
+
+*Note:* This specification is built upon [Capella](../../capella/beacon_chain.md) and is under active development.
+
+## Preset
+
+### Time parameters
+
+| Name | Value | Unit | Duration |
+| - | - | - |
+| `REUSE_VALIDATOR_INDEX_DELAY` | `uint64(2**16)` (= 65,536) | epochs | ~1 year |
```

Comment:
but `65536 * 32 * 12 / 60 / 60 / 24 = 291.2711111111111` :p

we used ~0.8 years for `EPOCHS_PER_HISTORICAL_VECTOR` 

```suggestion
| `REUSE_VALIDATOR_INDEX_DELAY` | `uint64(2**16)` (= 65,536) | epochs | ~0.8 year |
```